xServerWorker/QuartzJobConfigurator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Quartz;
+using Quartz.Impl;
+using xServerWorker.Jobs;
+
+namespace xServerWorker
+{
+    public static class QuartzJobConfigurator
+    {
+        private static StdSchedulerFactory factory = new StdSchedulerFactory();
+
+        public static async Task ConfigureJobsAsync()
+        {
+
+            IScheduler scheduler = await factory.GetScheduler();
+            await scheduler.Start();
+
+            await AddJobToScheduler<GraviexOrderbookJob>(scheduler, "0/10 0/1 * 1/1 * ? *");
+          //  await AddJobToScheduler<PayXServersJob>(scheduler, "0/10 0/1 * 1/1 * ? *");
```

Comment:
How many xServers will be running using this account?
Does each xServer need to setup their own Graviex Account?

Reply:
These are public endpoints